tests/Support/Constraint/StringContainsCss.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Tests\Support\Constraint;
+
+use PHPUnit\Framework\Constraint\Constraint;
+
+/**
+ * This constraint asserts that the string it is evaluated for contains some specific CSS, allowing for cosmetic
+ * whitespace differences.
+ *
+ * The CSS string passed in the constructor.
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ */
+class StringContainsCss extends Constraint
+{
+    use CssConstraint;
+
+    /**
+     * @var string
+     */
+    private $css;
+
+    /**
+     * @param string $css
+     */
+    public function __construct(string $css)
+    {
+        parent::__construct();
+
+        $this->css = $css;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function toString(): string
+    {
+        return 'contains CSS `' . $this->css . '`';
+    }
+
+    /**
+     * @inheritdoc
```

Comment:
Please no `@inheritdoc`. It's good for machines, but not very useful for the human reader.

Reply:
Fair enough.  It works fine when sent though something like phpDocumentor, for online documentation, but I guess that's not the most common use case (particularly for an internal class)!

Reply:
Still todo :/

Reply:
Now done.